tools/census_contrib/src/census_contrib/load.py

```diff
@@ -126,19 +126,19 @@ def __enter__(self) -> Self:
         self.joinids_sort = np.argsort(self.joinids)
         self.embeddings = np.load(self.embeddings_uri, mmap_mode="r")
 
-        self.n_obs = len(self.joinids)
+        self.n_embeddings = len(self.joinids)
```

Comment:
Nit: I feel like `n_embeddings` could be a bit confusing since embedding also refers to the matrix as a whole. Maybe `n_rows`?

Reply:
each row _is_ an embedding of one cell into the latent space.  Hence the name.

Reply:
Yeah, the name is indeed correct, but the term `embeddings` is overloaded and I feel like it might confuse the reader. That said, I don't think it really matters, so I'll resolve the conversation.